config/environments/development.rb

```diff
@@ -105,4 +105,9 @@
 
   # Annotate rendered view with file names.
   config.action_view.annotate_rendered_view_with_filenames = ENV.fetch('ANNOTATE_RENDERED_VIEWS', false)
+
+  config.hosts += [
```

Comment:
are these needed?

Reply:
seems like everything under .localhost should resolve to 127.0.0.1 - does cy.energysparks.localhost work on your machine?